docs/jobs-api.md

```diff
@@ -285,3 +285,20 @@ GET /api/v0/paths/forever.juttle HTTP/1.1
     }
 }
 ```
+
+###GET /version
```

Comment:
Should have noticed this in the original PR adding the route, but why not put this under `/api/v0/version`? 

Also I kinda think this should be named `/api/v0/version-info`.